packages/angular_devkit/build_angular/src/tools/sass/rebasing-importer.ts

```diff
@@ -74,6 +83,26 @@ abstract class UrlRebasingImporter implements Importer<'sync'> {
       updatedContents.update(start, end, rebasedUrl);
     }
 
+    for (const { start, end, value } of findImports(contents)) {
```

Comment:
Suggestion: Rename `value` to something more meaningful. Maybe `importSpecifier`?